test/EFCore.PG.FunctionalTests/Scaffolding/NpgsqlDatabaseModelFactoryTest.cs

```diff
@@ -1532,6 +1532,58 @@ public void Index_method()
                 @"DROP TABLE ""IndexMethod""");
         }
 
+        [Fact]
+        public void Index_operators()
+        {
+            Test(
+                @"
+CREATE TABLE ""IndexOperators"" (a text, b text);
+CREATE INDEX ix_with ON ""IndexOperators"" (a, b varchar_pattern_ops);
+CREATE INDEX ix_without ON ""IndexOperators"" (a, b);",
+                Enumerable.Empty<string>(),
+                Enumerable.Empty<string>(),
+                dbModel =>
+                {
+                    var table = dbModel.Tables.Single();
+
+                    var indexWith = table.Indexes.Single(i => i.Name == "ix_with");
+                    Assert.Equal(new[] { null, "varchar_pattern_ops" }, indexWith.FindAnnotation(NpgsqlAnnotationNames.IndexOperators).Value);
+
+                    var indexWithout = table.Indexes.Single(i => i.Name == "ix_without");
+                    Assert.Null(indexWithout.FindAnnotation(NpgsqlAnnotationNames.IndexOperators));
+                },
+                @"DROP TABLE ""IndexOperators""");
+        }
+
+        [Fact]
+        public void Index_covering()
+        {
+            if (Fixture.TestStore.GetPostgresVersion() < new Version(11, 0))
```

Comment:
Just a note, but should we at some point start testing against multiple release versions of PostgreSQL?

Might be something to consider in a few weeks, after 2.2 and the move to Azure Pipelines.

Reply:
A long, long time ago it actually worked this way. The problems were a much longer build time (multiplied by the number of versions supported), and significantly more complexity in maintenance of the CI platform (this may be gone with Azure, or not).

As a general rule, I'm not sure we've seen many version-related issues that weren't reported and fixed rather quickly. So while testing all versions is theoretically better, I'm simply not sure the added value is worth the downsides... Am definitely open to discussing this though - maybe open a new issue for discussing?